README.md

```diff
@@ -70,10 +70,10 @@ The project has a dependency on Ruby because it uses Jekyll. If you do not have
 
 ```shell
 curl -sSL https://get.rvm.io | bash -s stable --ruby
-rvm install 2.7
 brew install openssl
 brew link openssl --force
-gem install eventmachine -- --with-openssl-dir=/usr/local/opt/openssl@1.1
+rvm install ruby-2.7.8 --with-openssl-dir=$(brew --prefix openssl@1.1)
```

Comment:
Can we update this to `ruby-3.0.0`? That's what the GHA are using.

Reply:
After trial-and-error I bumped it to 3.2.3 which is the latest version that our machines and GHA comfortably support.